cloudconfig/templates.go

```diff
@@ -393,6 +393,49 @@ function Invoke-GarmFailure() {
 	}
 }
 
+function Get-SystemInfo {
```

Comment:
Just a small nit: Rename this to `Set-SystemInfo`. We're setting the system info in the GARM API, because we're sending the payload as part of this function. The `Get` verb is meant for functions that fetch information and present it as a return value. This sets something in a remote system. It never returns anything.